esp-hal/src/spi/master.rs

```diff
@@ -2232,102 +2238,40 @@ pub trait InstanceDma: Instance + DmaEligible {
             reg_block.dma_in_link().write(|w| w.bits(0));
         }
 
-        self.configure_datalen(rx_buffer.length(), tx_buffer.length());
+        let rx_len = rx_buffer.length();
+        let tx_len = tx_buffer.length();
+        self.configure_datalen(rx_len, tx_len);
 
         // re-enable the MISO and MOSI
```

Comment:
this comment should be removed now

Reply:
why?

Reply:
You've change the code in such a way that the comment is now wrong, "re-enable" doesn't make sense when there's no "disabling code" to be seen.
It's more like "set" or "configure" rather than "re-enable".